File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.swift
let input = readLine()!.split(separator: " ").map { Int($0)! }
var siteDictionary: Dictionary<String, String> = [String: String]()

for i in 0..<input[0] {
    let site = readLine()!.split(separator: " ")
    siteDictionary.updateValue(String(site[1]), forKey: String(site[0]))
}

for i in 0..<input[1] {
    let targetSite = readLine()!
    print(siteDictionary[targetSite, default: ""])
}

File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.py
import sys
input = sys.stdin.readline

n,m = map(int, input().split())
dic = {}

for _ in range(n):
    key, value = input().split()
    dic[key] = value
    
for _ in range(m):
    target = input().strip()
    print(dic[target])
